Makefile

```diff
@@ -32,7 +34,11 @@ cargo-sort:
 	cargo install cargo-sort
 	cargo sort -c -w
 
-check: check-fmt check-clippy cargo-sort
+fmt-toml:
+	cargo install taplo-cli --locked
+	$(foreach x, $(tomlfiles), $(shell taplo fmt $x))
```

Comment:
taplo supports formatting the entire directory, so there's no need for a foreach loop here.

Reply:
Good to know. Updated